pmoves/services/hi-rag-gateway-v2/tests/test_mindmap_route.py

```
@@ -205,6 +218,98 @@ def session(self):
+    fastapi_mod.WebSocketDisconnect = _WebSocketDisconnect
+    _install_stub("fastapi", fastapi_mod, stubs)
+
+    staticfiles_mod = types.ModuleType("fastapi.staticfiles")
+    staticfiles_mod.StaticFiles = type('StaticFiles', (), {'__init__': lambda *a, **kw: None})
+    _install_stub("fastapi.staticfiles", staticfiles_mod, stubs)
+
+    pydantic_mod = types.ModuleType("pydantic")
+
+    class _BaseModel:
+        def __init__(self, **data):
+            for k, v in data.items(): setattr(self, k, v)
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)